### ISSUE TITLE
Callback vs. Method

### ISSUE THREAD

**Issue by AlinaTaoRao** [lab-antwerp-1/home#255]
1. What's the difference between  `callback function` and `method`? For me, callback function and method are playing the same role. I guess there is at least a situation that callback will stands out, but what? when?

**Comment by gelilaa** [comment]
@AlinaTaoRao you are right, they are both set of instructions to perform a task. the real difference is . that a method is associated with an object, while a function is not.

**Comment by AlinaTaoRao** [comment]
@gelilaa 
Thanks for your answer. 😄

**Comment by colevandersWands** [comment]
@AlinaTaoRao , both are functions functions the difference is how they are used.  Like @gelilaa said:

- a method is associated with an object (or array) and has a `.` before it - `[].reverse()`
- a callback is a stand-alone function that is passed as an argument:
  ```js
  const aCallback = () => {};
  [].map(aCallback);  // .map is the method, aCallback is the callback
  ```
